client/post-editor/post-editor.jsx

```diff
@@ -1384,48 +1371,58 @@ export const PostEditor = createReactClass( {
 	},
 } );
 
-export default connect(
-	state => {
-		const siteId = getSelectedSiteId( state );
-		const postId = getEditorPostId( state );
-		const userId = getCurrentUserId( state );
+const enhance = flow(
+	localize,
+	protectForm,
+	connect(
+		state => {
+			const siteId = getSelectedSiteId( state );
+			const postId = getEditorPostId( state );
+			const userId = getCurrentUserId( state );
 
-		return {
-			siteId,
-			postId,
-			selectedSite: getSelectedSite( state ),
-			selectedSiteDomain: getSiteDomain( state, siteId ),
-			editorModePreference: getPreference( state, 'editor-mode' ),
-			editorSidebarPreference: getPreference( state, 'editor-sidebar' ) || 'open',
-			editPath: getEditorPath( state, siteId, postId ),
-			edits: getPostEdits( state, siteId, postId ),
-			dirty: isEditedPostDirty( state, siteId, postId ),
-			hasContent: editedPostHasContent( state, siteId, postId ),
-			layoutFocus: getCurrentLayoutFocus( state ),
-			hasBrokenPublicizeConnection: hasBrokenSiteUserConnection( state, siteId, userId ),
-			isSitePreviewable: isSitePreviewable( state, siteId ),
-			isEditorOnlyRouteInHistory: isEditorOnlyRouteInHistory( state ),
-			isConfirmationSidebarEnabled: isConfirmationSidebarEnabled( state, siteId ),
-		};
-	},
-	dispatch => {
-		return bindActionCreators(
-			{
-				setEditorLastDraft,
-				resetEditorLastDraft,
-				receivePost,
-				editPost,
-				savePostSuccess,
-				setEditorModePreference: savePreference.bind( null, 'editor-mode' ),
-				setEditorSidebar: savePreference.bind( null, 'editor-sidebar' ),
-				setLayoutFocus,
-				setNextLayoutFocus,
-				saveConfirmationSidebarPreference,
-				recordTracksEvent,
-			},
-			dispatch
-		);
-	},
-	null,
-	{ pure: false }
-)( protectForm( localize( PostEditor ) ) );
+			return {
+				siteId,
+				postId,
+				selectedSite: getSelectedSite( state ),
+				selectedSiteDomain: getSiteDomain( state, siteId ),
+				editorModePreference: getPreference( state, 'editor-mode' ),
+				editorSidebarPreference: getPreference( state, 'editor-sidebar' ) || 'open',
+				editPath: getEditorPath( state, siteId, postId ),
+				edits: getPostEdits( state, siteId, postId ),
+				dirty: isEditedPostDirty( state, siteId, postId ),
+				hasContent: editedPostHasContent( state, siteId, postId ),
+				layoutFocus: getCurrentLayoutFocus( state ),
+				hasBrokenPublicizeConnection: hasBrokenSiteUserConnection( state, siteId, userId ),
+				isSitePreviewable: isSitePreviewable( state, siteId ),
+				isEditorOnlyRouteInHistory: isEditorOnlyRouteInHistory( state ),
+				isConfirmationSidebarEnabled: isConfirmationSidebarEnabled( state, siteId ),
+				nestedSidebarTarget: getNestedSidebarTarget( state ),
+			};
+		},
+		dispatch => {
```

Comment:
This can probably be turned into an object.
```jsx
{
    setEditorLastDraft,
    //...
    setEditorModePreference: partial( savePreference, 'editor-mode' ),
```

Reply:
I'm definitely for doing this, as it stands we'd be creating new functions (by `.bind`ing) repeatedly (details @ #19320).
Instead of adding to the scope of this PR (since it's whitespace changes only on these lines), I've opened a new PR @ #19588 :)